### ISSUE TITLE
[Jetengine plugins family] Swiper Library Switch 🔄️

### ISSUE THREAD

**Issue by nreljed** [CrocoBlock/suggestions#6432]
Hello , 
we will soon have performance and speed problems and unnecessary additional code with jetengine 😅 ... 
Elementor 3.11 says so !
@MjHead can you enable red alert and project code "Swiper Library Switch"?
I think now is the time to do the great migration 🚗

I think it's the perfect time to start the switchover don't we let jetengine carry the heavy burden of slick.js! 💣
Let's hold our hands to the future and beyond
I'm not going to detail everything why do this, because I already did it a year ago in an old post

https://github.com/Crocoblock/suggestions/issues/4622

Good reading

![Capture d’écran 2023-01-25 090907](https://user-images.githubusercontent.com/82481592/214513709-b1b668aa-4b0b-41c1-bd0f-0066b267bf9e.png)
![Capture d’écran 2023-01-25 092147](https://user-images.githubusercontent.com/82481592/214513714-2110d77e-efe3-4e2b-b388-89f66f45fa3f.png)

**Comment by Michaelgimii** [comment]
Hi @nreljed 
Do you mean that with the update of swiper js by Elementor, the performance of jet plugins will decrease?

**Comment by nreljed** [comment]
> Hi @nreljed Do you mean that with the update of swiper js by Elementor, the performance of jet plugins will decrease?

Yes,  inevitably we will have two libraries so more script files to load so performance will inevitably drop
not to mention that if elementor already does better with swiper, it will be very delicate to explain why jetengine does not use the swiper library...
it's time to ditch the (already dead) slick.js and switch to swiper.js
You can see the difference on the repos, no need to explain more 😀

[swiper.js](https://github.com/nolimits4web/swiper)
![Capture d’écran 2023-01-25 104629](https://user-images.githubusercontent.com/82481592/214533353-8e1dacb1-854a-4d7b-a4f7-141c96a56add.png)

[slick.js](https://github.com/kenwheeler/slick)
![Capture d’écran 2023-01-25 104643](https://user-images.githubusercontent.com/82481592/214533378-2a916d9f-2f1a-4769-ac7b-391e41c30a57.png)

**Comment by Michaelgimii** [comment]
oh, I thought crocoblock uses Elementor's swiper js. Because I know that jet woobuilder uses Elementor's swiper.
So it would be great if Jet Engine uses Elementor's swiper (8.4.5). Carousels will now be expandable better than before.

**Comment by nreljed** [comment]
> oh, I thought crocoblock uses Elementor's swiper js. Because I know that jet woobuilder uses Elementor's swiper. So it would be great if Jet Engine uses Elementor's swiper (8.4.5). Carousels will now be expandable better than before.

This is very important information that you reveal to us 
it means that the person(s) behind jetwoobuilder has started the transition !
but we don't really know who manages who so we will assume the fact that @MjHead 
is the person responsible for deciding this transition for jetengine and other plugins ?
maybe it's already in progress ?  let's wait for a reaction to know more

**Comment by Michaelgimii** [comment]
If I'm not mistaken, @Floydtm manages crocoblock plugins.

**Comment by 79ho3ein** [comment]
+1

**Comment by morosmo** [comment]
+1

**Comment by beposss** [comment]
+1 
That's very important

**Comment by SolunaCG** [comment]
+1 any update here? Coming from #6899 a better Slider would be highly appreciated! The Freemode that swiper.js offers is something that I´d love to have with the JetEngine Slider, too!

https://swiperjs.com/demos#freemode

**Comment by Studio21Creative** [comment]
Hi Crocoblock, it’s coming up to 2 years on this issue. Do you possibly have an update for us? If it’s something you never think you’ll implement please also let us know and I’ll look at alternatives. Cheers

**Comment by aalexandr21** [comment]
In JetEngine 3.8.6, an option was added for choosing the Library 
<img width="1727" height="703" alt="Image" src="https://github.com/user-attachments/assets/d39fda46-87b6-4c89-9453-b5288dd9d249" />

❗❗❗**This is the first step for moving to the Swiper library, so please check if everything is OK with your listings after changing the library.**

**Comment by Studio21Creative** [comment]
Thank you guys - looks great to me!
